### DISCUSSION TITLE
Discharge summaries during ED

### DISCUSSION THREAD

**Discussion by abhinayvyas** [MIT-LCP/mimic-code#1516] [MIMIC-IV-ED] [answered]
While MIMIC-IV note contains free text about patients' inpatient and ED stay, is there any data summarising patient journey during ED?

**Comment by alistairewj** [comment, marked as answer]
Not yet. ED notes are a type of progress note which are not deidentified yet.

**Reply by abhinayvyas** [reply]
Thanks for the prompt reply @alistairewj, please let me know if I could contribute in any way. I have written a few de-identification algorithms that remove PHI.

**Reply by alistairewj** [reply]
So have I! ;) Thanks for the offer to help, sincerely. Unfortunately it's often too difficult to work out the administrative hoops. The main thing we need to do is run our existing algorithms and validate them. We are in the midst of that now (and unsurprisingly enough we are on the administrative approvals part).

**Reply by abhinayvyas** [reply]
I understand, thanks again!
